### PR TITLE
[14.0][MIG] web_tree_image_tooltip

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration.
Screenshot from 2021-03-27 17-12-28

cc @kittiu @Saran440 @ps-tubtim

### CODE REVIEW

**Review by ioans73 — APPROVED** [review]
Functional review

**Review by ps-tubtim — APPROVED** [review]
Functional Test: Ok.

Peek 2021-03-29 16-41

**Comment by claudiagn** [comment]
LGTM!

**Review by claudiagn — APPROVED** [review]
LGTM! :+1:

**Comment by newtratip** [comment]
@OCA/web-maintainers, Could you review/merge this ?

File: web_tree_image_tooltip/README.rst [hunk 1]
```diff
@@ -72,6 +72,7 @@ Contributors
 * Meet Dholakia <m.dholakia.serpentcs@gmail.com>
 * Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
 * Phuc Tran Thanh <phuc@trobz.com>
+* Tharathip Chaweewongphan <tharathipc@ecosoft.co.th>
```
Comment (simahawk) [inline web_tree_image_tooltip/README.rst:75]: For plain migrations that do not require substantial changes, normally we don't add ourselves to contributors.
Now, I see that this module didn't follow the rule in the past as well. Not blocking...
Just keep it in mind for the future :wink:

**Review by simahawk — COMMENTED** [review]
/ocabot merge nobump

**Pull request merged**